### DISCUSSION TITLE
Localhost Loopback missing UDP packets

### DISCUSSION THREAD

**Discussion by qichenxu** [chmorgan/sharppcap#429] [Q&A]
Hi, I'm using the queueing method for background processing some UDP packets I'm sending from a localhost test program. What I noticed is that the captured packets would be less than what I sent out (in the UDP payload I also included a counter for each message sent). I constantly got missing packets when sent anything quicker than 1s. 
I tried to play with the sleep duration but with no success. Is it related to performance limitation to the localhost capturing? How should I debug which part of the code is causing such missing captured packets?

**Comment by kayoub5** [comment]
try to increase the buffer size (both the pcap buffer size and udp socket buffer size) and use a pcap filter if possible.
